### PR TITLE
Update test questions and clarify instructions

### PR DESCRIPTION
**Pull request by martynamaron**
I have:
1. Added "passing all tests" to the end goal
2. Made all question instructions clearer
3. Removed Q7 (will be covered in a separate test for JS2) and Q10 (unclear and too complicated)
4. Removed the loop suggestion from Q4, as we encourage using .map() and .forEach()

### CODE REVIEW

File: module1.js [hunk 1]
```diff
@@ -23,179 +23,106 @@ function question1(name1) {
...
 
 /**
-  +++++++++++++++++++++++++++++++++++++++++++
   Question 4:
-  Write a for loop that loops over the numbers from 0 to 100, returning an array containing only every second number. So it should return [ 0, 2, 4, 6, 8, 10 .... etc.]
+  Write a function that takes an array of integers as input, and returns an array containing only every second element.
 
-  @param none
-  @returns [array 0 ... 100] 
+  Use the .forEach() or .map() functions to do this.
```
Comment (alismith-dev) [inline module1.js]: Hmm, not that I look at this again, `.filter` would be the best choice. Not sure how to word this without directly mentioning that. Maybe that's not a bad thing?

File: module1.js [hunk 2]
```diff
@@ -23,179 +23,106 @@ function question1(name1) {
...
   Question 2:
-  IF / ELSE Statements:
-  Check if two numbers are equal, if they are return true or otherwise return false
-  @param {number} number1 a regular integer
-  @param {number} number2 a regular integer
-  @returns {boolean} true or false
-*/
+  Write a function that compares two input integers.
 
-// function question2(number1, number2) {
```
Comment (alismith-dev) [inline module1.js:37]: If we're uncommenting the question functions we might as well uncomment them in `module.exports` too. The tests should still fail (as all of the functions are no-ops)

File: module1.js [hunk 3]
```diff
@@ -23,179 +23,106 @@ function question1(name1) {
...
-
+function question4(array) {
+  // TODO:
+}
 
 /**
-  +++++++++++++++++++++++++++++++++++++++++++
   Question 5:
-  Write a function called multiply that multiplies two numbers together. It should take two numbers as parameters and return the answer.
+  Write a function that multiplies two input integers or two input floats (positive or negative).
```
Comment (alismith-dev) [inline module1.js]: I think it might be confusing to mention integers and floats. In terms of this question there's no real difference

**Comment by alismith-dev** [comment]
Looks great!

File: module1.js (hunk 1 above)
Comment (textbook) [reply module1.js]: We could refer to "iteration methods" in general: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array#Iteration_methods

File: module1.js (hunk 2 above)
Comment (martynamaron) [reply module1.js:37]: Done - I removed the instruction to uncomment the module.exports as well

File: module1.js (hunk 3 above)
Comment (martynamaron) [reply module1.js]: Done.
I also realised that the test tries to multiply `0.7` and `2.3`, if you just do `0.7 * 2.3` the result is 1.6099999 - and the test expects `1.61` - do you think that could be a problem?

File: module1.js (hunk 1 above)
Comment (martynamaron) [reply module1.js]: Done

**Comment by martynamaron** [comment]
I've also fixed some of the instructions to not include "uncommenting functions and module.exports" and added forking and cloning the repo to the readme

File: module1.js (hunk 3 above)
Comment (alismith-dev) [reply module1.js]: Yay for JavaScript floats! 🎉 I can only imagine that was intentional to test for knowledge about this. However I don't think we covered that in the syllabus (you probably know better than I do), so I'd be inclined to remove it. I don't necessarily think it's core to JS knowledge either.

**Comment by alismith-dev** [comment]
I think if we remove that test about floats, this is good to go! Thanks a lot :)

**Comment by alismith-dev** [comment]
Merging. Thanks :) And congrats on your first github PR 🎆 🍰

**Pull request merged**